datumaro/plugins/sly_pointcloud_format/extractor.py

```diff
@@ -121,7 +122,11 @@ def _parse_tag(tag):
                 label = categories[AnnotationType.label].find(obj["classTitle"])[0]
 
                 attributes = {}
-                attributes["track_id"] = obj["id"]
```

Comment:
If we add ``track_id`` as an attribute, CVAT code consider the object to be a track. Even if the file initially was exported without any tracks. 

So, I put it under condition and it will be only read if "keyframe" attribute is also dumped. "keyframe" is written by CVAT only for tracks.